h/migrations/versions/8b4b4fdef955_auto_vacuum_annotation.py

```diff
@@ -0,0 +1,30 @@
+"""Change auto-vacuum settings on big tables."""
+from alembic import op
+
+revision = "8b4b4fdef955"
+down_revision = "0de98307b3c0"
+
+BIT_TABLES = [
+    "annotation",
+    "annotation_slim",
+    "annotation_metadata",
+    "document",
+    "user",
+    "group",
+]
+
+
+def upgrade():
+    for table in BIT_TABLES:
+        # Set auto vacuum to kick in after 5% of dead tuples (vs total in the table) are detected
```

Comment:
What is the current percentage of dead tuples in the live prod DB?

Reply:
20%

Reply:
In which table? Around ~10% in the annotation table, see: https://github.com/hypothesis/h/issues/8279